specs/protocol/isthmus/derivation.md

```diff
@@ -314,29 +314,25 @@ cast keccak $(cast concat-hex 0x000000000000000000000000000000000000000000000000
 ## EIP-2935 Contract Deployment
 
 [EIP-2935](https://eips.ethereum.org/EIPS/eip-2935) requires a contract to be deployed. To deploy this contract,
-a deposit transaction is created with attributes matching the EIP:
+a type 0 (legacy) transaction is created with attributes matching the EIP:
 
-- `from`: `0xE9f0662359Bb2c8111840eFFD73B9AFA77CbDE10`
-- `to`: `null`,
-- `mint`: `0`
-- `value`: `0`
-- `gasLimit`: `250,000`
-- `data`: `0x60538060095f395ff33373fffffffffffffffffffffffffffffffffffffffe14604657602036036042575f35600143038111604257611fff81430311604257611fff9006545f5260205ff35b5f5ffd5b5f35611fff60014303065500`,
-- `sourceHash`: `0xbfb734dae514c5974ddf803e54c1bc43d5cdb4a48ae27e1d9b875a5a150b553a`
-  computed with the "Upgrade-deposited" type, with `intent = "Isthmus: EIP-2935 Contract Deployment"
-
-This results in the EIP-2935 contract being deployed to `0x0F792be4B0c0cb4DAE440Ef133E90C0eCD48CCCC`, to verify:
-
-```bash
-cast compute-address --nonce=0 0xE9f0662359Bb2c8111840eFFD73B9AFA77CbDE10
-Computed Address: 0x0F792be4B0c0cb4DAE440Ef133E90C0eCD48CCCC
-```
+- `type`: `0x0`
+- `nonce`: `0x0`
+- `to`: `null`
+- `gas`: `0x3d090`,
+- `gasPrice`: `0xe8d4a51000`
+- `value`: `0x0`
+- `data`: `0x60538060095f395ff33373fffffffffffffffffffffffffffffffffffffffe14604657602036036042575f35600143038111604257611fff81430311604257611fff9006545f5260205ff35b5f5ffd5b5f35611fff60014303065500`
+- `v`: `0x1b`
+- `r`: `0x539`
+- `s`: `0xaa12693182426612186309f02cfe8a80a0000`
```

Comment:
```suggestion
- `v`: `0x1b`
- `r`: `0x539`
- `s`: `0xaa12693182426612186309f02cfe8a80a0000`
```

We deploy this contract via a special deposit transaction inserted by the attributes builder in the transition block. It can be deployed via a legacy transaction as L1 specifies, but doesn't need to be. See [here](https://github.com/op-rs/kona/blob/e39a82a8310a0c1067414c4a638cc309d83a19aa/crates/protocol/hardforks/src/isthmus.rs#L221-L230) in `kona`'s upgrade transaction definitions, and [here](https://sepolia-optimism.etherscan.io/tx/0xf3ddde30514a8d00b0b92f14b9ce6b3170ea30f5f45b1afdc0905801134ffefe) where it was deployed via a special system transaction on OP Sepolia.

For the sake of the Isthmus derivation spec, we should retain the deployment as a special upgrade deposit tx.